flink-docs/src/test/java/org/apache/flink/docs/configuration/ConfigOptionsDocsCompletenessITCase.java

```diff
@@ -257,7 +254,7 @@ private static void compareDocumentedAndExistingOptions(
                             // Use split to exclude document key tag.
                             String key = tableRow.child(0).text().split(" ")[0];
                             String defaultValue = tableRow.child(1).text();
-                            String typeValue = tableRow.child(2).text();
+                            String typeValue = tableRow.child(2).html();
```

Comment:
Here we're parsing the type from the current docs, and "on the other side" we're generating it from the `ConfigOption`. Since there we do so by calling `typeToHtml`, we need to also preserve the HTML here for the comparison to work.

This happened to work previously because `#text()` preserves HTML in `<p>A</p>B`, but not in `<p>A</p>` (for whatever reason).